src/components/application_manager/test/policy_handler_test.cc

```diff
@@ -176,7 +176,8 @@ class PolicyHandlerTest : public ::testing::Test {
     ON_CALL(policy_settings_, enable_policy()).WillByDefault(Return(true));
     ON_CALL(app_manager_, event_dispatcher())
         .WillByDefault(ReturnRef(mock_event_dispatcher_));
-    std::string path = file_system::CreateDirectory("storage");
+    std::string path("storage");
+    file_system::CreateDirectory(path);
```

Comment:
@SKobziev same as above

Reply:
@AByzhynar It is corrected. Could you please check it?